client/blocks/login/style.scss

```diff
@@ -72,3 +71,11 @@
 	display: block;
 	margin-bottom: 16px;
 }
+
+.two-factor-authentication__form-button {
+	width: 100%;
+}
+
+.card.two-factor-authentication__form-action {
+	background: lighten( $gray, 34% );
```

Comment:
Is there a good reason why this is **34%**? Seems unnecessary when we have `$gray-light`, which is **33%**